### PR TITLE
refactor modules of cce

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
The following resources are refactored:
hwc_cce_cluster
hwc_cce_node

### CODE REVIEW

File: library/hwc_cce_cluster.py [hunk 1]
```diff
@@ -0,0 +1,1217 @@
...
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: hwc_ecs_instance
+description:
+    - instance management.
+short_description: Creates a resource of Ecs/Instance in Huawei Cloud
```
Comment (niuzhenguo) [inline library/hwc_cce_cluster.py:24]: ditto.

File: library/hwc_cce_node.py [hunk 2]
```diff
@@ -0,0 +1,1544 @@
...
+# Documentation
+###############################################################################
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: hwc_ecs_instance
```
Comment (niuzhenguo) [inline library/hwc_cce_node.py:21]: should be cce node instead of ecs

File: library/hwc_cce_node.py [hunk 3]
```diff
@@ -0,0 +1,1544 @@
...
+description:
+    - instance management.
+short_description: Creates a resource of Ecs/Instance in Huawei Cloud
```
Comment (niuzhenguo) [inline library/hwc_cce_node.py:24]: ditto.

File: library/hwc_cce_cluster.py [hunk 4]
```diff
@@ -0,0 +1,1217 @@
...
+# Documentation
+###############################################################################
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: hwc_ecs_instance
```
[the remark this answers is not in the document]
Comment (Jason-Zhang9309) [reply library/hwc_cce_cluster.py]: fixed

File: library/hwc_cce_cluster.py (hunk 1 above)
Comment (Jason-Zhang9309) [reply library/hwc_cce_cluster.py]: fixed

File: library/hwc_cce_node.py (hunk 2 above)
Comment (Jason-Zhang9309) [reply library/hwc_cce_node.py]: fixed

File: library/hwc_cce_node.py (hunk 3 above)
Comment (Jason-Zhang9309) [reply library/hwc_cce_node.py]: fixed

**Pull request merged**